valve-monitor/web-app/src/pages/index.tsx

```diff
@@ -51,37 +72,59 @@ const Home: NextPage<HomeData> = ({ valveMonitorConfig, err }) => {
-          dangerouslySetInnerHTML={{ __html: err }}
+          dangerouslySetInnerHTML={{ __html: err || error }}
         />
       ) : (
         <LoadingSpinner isLoading={isLoading}>
           <div>
             {isErrored && (
               <Alert type="error" message={errorMessage} closable />
             )}
-            <h3 className={styles.sectionTitle}>Fleet Controls</h3>
-            <Row gutter={16}>
-              <Col className={styles.motionFrequencyCard}>
-                <MonitorFrequencyCard
-                  currentFrequency={monitorFrequency}
-                  setCurrentFrequency={setMonitorFrequency}
-                  setErrorMessage={setErrorMessage}
-                  setIsErrored={setIsErrored}
-                  setIsLoading={setIsLoading}
-                />
-              </Col>
-              <Col>
-                <AlarmThresholdCard
-                  currentMinFlowThreshold={minFlowThreshold}
-                  currentMaxFlowThreshold={maxFlowThreshold}
-                  setCurrentMinFlowThreshold={setMinFlowThreshold}
-                  setCurrentMaxFlowThreshold={setMaxFlowThreshold}
-                  setIsErrored={setIsErrored}
-                  setIsLoading={setIsLoading}
-                  setErrorMessage={setErrorMessage}
-                />
-              </Col>
-            </Row>
+
+            <div>
+              {valveMonitorDevices && (
+                <>
+                  <h3 className={styles.sectionTitle}>Fleet Controls</h3>
+                  <Row gutter={16}>
+                    <Col className={styles.motionFrequencyCard}>
+                      <MonitorFrequencyCard
+                        currentFrequency={monitorFrequency}
+                        setCurrentFrequency={setMonitorFrequency}
+                        setErrorMessage={setErrorMessage}
+                        setIsErrored={setIsErrored}
+                        setIsLoading={setIsLoading}
+                      />
+                    </Col>
+                    <Col>
+                      <AlarmThresholdCard
+                        currentMinFlowThreshold={minFlowThreshold}
+                        currentMaxFlowThreshold={maxFlowThreshold}
+                        setCurrentMinFlowThreshold={setMinFlowThreshold}
+                        setCurrentMaxFlowThreshold={setMaxFlowThreshold}
+                        setIsErrored={setIsErrored}
+                        setIsLoading={setIsLoading}
+                        setErrorMessage={setErrorMessage}
+                      />
+                    </Col>
+                  </Row>
+                  <Row gutter={[16, 24]}>
+                    <Col span={24}>
+                      <div className={styles.tableHeaderRow}>
+                        <h3 className={styles.sectionTitle}>
+                          Individual Controls
+                        </h3>
+                      </div>
+                    </Col>
+                    <Col xs={12} sm={7} md={6} lg={4} />
```

Comment:
Is this necessary?

Reply:
Nope. I have no idea why it's there. It may become necessary when the alarms dismiss button is added but we'll cross that bridge later